src/ResourceManager/Sql/Commands.Sql/Replication/Model/AzureReplicationLinkModel.cs

```diff
@@ -41,6 +42,11 @@ public class AzureReplicationLinkModel : AzureSqlDatabaseReplicationModelBase
         /// </summary>
         public string SecondaryServiceObjectiveName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the requested Sku for the partner
+        /// </summary>
+        public Sku SecondarySku { get; set; }
```

Comment:
Can flatten here?

Reply:
Also can you paste what the output of Get-AzureRmSqlDatabaseReplicationLink looks like?

Reply:
It is not displayed because of filter in Types.ps1xml file. However I think that we are still not supposed to directly expose the SDK types.

Reply:
@jaredmoo The get replication link looks like"
LinkId                   : d8a67cfe-10dd-4e67-9066-63a710d99571
ResourceGroupName        : payiNothEuropeProdResourceGroup
ServerName               : payi-svr1
DatabaseName             : db1copy
Role                     : Primary
Location                 : Southeast Asia
PartnerResourceGroupName : payiNothEuropeProdResourceGroup
PartnerServerName        : payi-svr1-geo
PartnerRole              : Secondary
PartnerLocation          : North Europe
AllowConnections         : All
ReplicationState         : CATCH_UP
PercentComplete          : 100
StartTime                : 5/14/2018 9:49:52 PM